src/ServiceFabric/ServiceFabric/Commands/NewAzureRmServiceFabricCluster.cs

```diff
@@ -160,7 +160,7 @@ public override string KeyVaultResouceGroupLocation
         [Parameter(Mandatory = false, ValueFromPipeline = true, ParameterSetName = ByDefaultArmTemplate,
               HelpMessage = "Azure key vault resource group name, if not given it will be defaulted to resource group name")]
         [ValidateNotNullOrEmpty]
-        public override string KeyVaultResouceGroupName { get; set; }
+        public override string KeyVaultResourceGroupName { get; set; }
```

Comment:
@charotAmine to avoid breaking changes, we need to add an alias to `KeyVaultResouceGroupName` here